sdk/netapp/Azure.ResourceManager.NetApp/CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Release History
 
 ## 1.4.0-beta.1 (Unreleased)
-
+- Updated to support ANF api-version 2022-09-01
```

Comment:
This same line has been added for 1.1.0 and 1.3.0.
If this intends to describe the api-verison, should it be 2023-05-01 for the current release?